app/containers/Landing/Landing.js

```diff
@@ -62,7 +56,8 @@ const mapStateToProps = createStructuredSelector({
 
 function mapDispatchToProps(dispatch) {
   return {
-    fetchCompany: (companyId) => dispatch(fetchCompanyInfo(companyId)),
+    goToTable: (companyId, tableNumber) => (dispatch(push(`company/${companyId}/table/${tableNumber}`)))
```

Comment:
Showing the company name instead of the id is more illustrative. And the route "/table" is needless in this case.
You could get the companyName related to this companyId and change to:

dispatch(push(${companyName}/table-${tableNumber}`

Reply:
I dont like the idea of `table-1`because it makes it harder to use the url to get such a paramenter.

assuming we want to fetch something based on the url, we would have to get the param `table-1` and then split it to get just the `1`.

In terms of adding the companyName to the url, I think this makes sense, I will make this change.

Reply:
Great!